cloudvolume/storage/storage_interfaces.py

```diff
@@ -190,10 +190,8 @@ def put_file(self, file_path, content, content_type, compress, cache_control=Non
     key = self.get_path_to_file(file_path)
     blob = self._bucket.blob( key )
 
-    # keep default gzip
-    if compress == "br":
-      blob.content_encoding = "br"
-    elif compress:
+    # gcloud disable brotli until content-encoding works
```

Comment:
I don't think this is the right patch b/c (a) we only have a problem with decompression, not compression (b) if `br` is selected for one file type this will transparently change to `gzip` without notifying the programmer something weird happened. I was thinking more along the lines of, perhaps we should raise an exception in CloudVolume if `compress='br'` for example. What do you think?

Reply:
yes, much better idea.